syntax.php

```diff
@@ -99,6 +99,11 @@ private function _getDefaultOptions(){
     function render($mode, &$renderer, $data) {
         global $conf;
 
+        $cache = $this->_cacheOption();
```

Comment:
It's a good idea to add a function in order to avoid putting too much responsibility in the `render` one.

I guess we could go even further, and put all this implementation of this new feature in an ad hoc method. So `render` could look like

    global $conf

    $this->_deactivateTheCacheIfNeeded(); //or any other more relevant name

    if ( $data['useLegacySyntax'] ){
    // and so on...